### PR TITLE
Avatar&badge

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review and provide your valuable feedback.

### CODE REVIEW

**Comment by gadgetfather** [comment]
make badges a little smaller in the badge on the avatar.  Class names are written neatly. Just at one place change it from "avatar" to "avatar-img"
Everything else looks good!

**Comment by ShrutiLakhani** [comment]
> make badges a little smaller in the badge on the avatar. Class names are written neatly. Just at one place change it from "avatar" to "avatar-img" Everything else looks good!

Made the changes. Thank You!

**Review by gadgetfather — APPROVED** [review]
works great now !

File: Avatar/Avatar.css
Comment (muskanbagrecha) [inline Avatar/Avatar.css]: Add display flex above other flex properties. Makes the code easier to read.

File: Avatar/Avatar.css
Comment (muskanbagrecha) [inline Avatar/Avatar.css]: Try not to give width and height values in px.

File: Avatar/Badge/Badge.css
Comment (deekshithmd) [inline Avatar/Badge/Badge.css]: I feel these indentation not necessary, and then everything looks good for me

File: Avatar/Badge/Badge.html
Comment (rahulyadav139) [inline Avatar/Badge/Badge.html]: You could provide all imports (external) in main component library CSS file rather than in an individual file.

File: Avatar/Badge/Badge.css
Comment (ShrutiLakhani) [reply Avatar/Badge/Badge.css]: Resolved!

File: Avatar/Avatar.css
Comment (ShrutiLakhani) [reply Avatar/Avatar.css]: Made the Changes. thanks!

File: Avatar/Avatar.css
Comment (ShrutiLakhani) [reply Avatar/Avatar.css]: I have added them in alphabetical order. Hoping its not an issue. Please approve.

File: Avatar/Badge/Badge.html
Comment (ShrutiLakhani) [reply Avatar/Badge/Badge.html]: I will do it once I merge everything to dev. Please approve, if we are good to go

File: Avatar/Badge/Badge.css
Comment (ShrutiLakhani) [reply Avatar/Badge/Badge.css]: Please approve

**Review by rahulyadav139 — APPROVED** [review]
Great you have made the necessary changes. Everything is great now. Go ahead to merge the branch.

**Pull request merged**
